acceptance/upgrade/upgrade_test.go

```diff
@@ -49,9 +49,9 @@ var _ = Describe("Epinio upgrade with running app", func() {
 
 	AfterEach(func() {
 		// After upgrade ...
-		env.DeleteApp(appName)
-		env.DeleteApp(appAfter)
-		env.DeleteService(service)
+		// env.DeleteApp(appName)
+		// env.DeleteApp(appAfter)
+		// env.DeleteService(service)
```

Comment:
Remove these lines, and also as before it's probably better to put everything in a `DeferCleanup`.